clang/test/CIR/CodeGen/libc.c

```diff
@@ -19,3 +19,21 @@ float testFabsf(float x) {
   return fabsf(x);
   // CHECK: cir.fabs %{{.+}} : !cir.float
 }
+
+int abs(int);
+int testAbs(int x) {
+  return abs(x);
+  // CHECK: cir.abs %{{.+}} : !s32i
```

Comment:
Please also check for the final LLVM output in this file. You can change this test and add an extra RUN line, just like in `clang/test/CIR/CodeGen/abstract-cond.c`. That way you don't need to necessarily add the test in `clang/test/CIR/Lowering/libc.cir` (those are usually hard to update and won't really do much in this case since the attribute isn't lowered).